source/events/events.rst

```diff
@@ -322,16 +326,20 @@ CTA and discussion, and then a proposal for a specification.
 
 .. _iact-events-obs-mode:
 
-OBS_MODE 
+OBS_MODE
 ~~~~~~~~
 
-The observation mode ``OBS_MODE`` is currently provenance information, not used
-by science tools to decide how to analyse the data. There is no set of defined
-modes yet. Thus, at the moment it is optional.
-
-Just to give an example: in H.E.S.S. the values of "WOBBLE" for wobble
-observations (pointing slightly off target) and "SCAN" for Galactic plane survey
-observation on a grid of sky positions (not wrt. a specific target) is used.
+The keyword ``OBS_MODE`` specifies the mode of the observation.
+This is used in cases where the instrument or detector can be configured to
+operate in different modes which significantly affect the resulting data, or to accomodate
+different types of insrument. More details can be found
+`here <https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/general/ogip_94_001/ogip_94_001.html>`__
+
+Additionally to the OGIP-defined values (``POINTING``, ``RASTER``, ``SLEW`` and ``SCAN``), we
+define an additional option ``DRIFT`` to accomodate ground-based instruments, in
```

Comment:
Remove additional (repetition).

Also: are IACTs not "ground-based"?

Reply:
ground-based wide-field instruments?

Reply:
static wide field ?

Reply:
I would support "ground-based wide-field" :+1: